app/database/queries/breeze_buddy/call_execution_config.py

```
@@ -246,3 +263,47 @@ def calling_activation_for_merchant_query(
+    Returns all unique shop_identifier values from call_execution_config.
+    Each shop_identifier represents a distinct merchant in the system.
+
+    Returns:
+        Tuple of (query string, empty values list)
+    """
+    query = f"""
+        SELECT DISTINCT shop_identifier
+        FROM {CALL_EXECUTION_CONFIG_TABLE}
+        WHERE shop_identifier IS NOT NULL
+        ORDER BY shop_identifier ASC
+    """
```

Ruff (0.14.10)

Possible SQL injection vector through string-based query construction

(S608)